packages/framework/build.xml

```diff
@@ -677,6 +677,20 @@
         </exec>
     </target>
 
+    <target name="product-search-create-structure-without-aliases" description="Creates structure without index aliases for searching via elasticsearch.">
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
+            <arg value="${path.bin-console}"/>
+            <arg value="shopsys:product-search:create-structure-without-aliases"/>
+        </exec>
+    </target>
+
+    <target name="product-search-create-aliases" description="Creates index aliases for current elasticsearch index.">
```

Comment:
I think that `product-search-create-structure` target should call this target too

Reply:
I don't really like it. Maybe if `product-search-create-structure` called create without alias and then create alias but I think that it would just be more confusing